Tests/SpotifyWebAPIMainTests/API Tests/SpoifyAPIArtistTests.swift

```diff
@@ -547,6 +546,14 @@ extension SpotifyAPIArtistTests {
             description: "testArtistTopTracks"
         )
         
+        Self.spotify.authorizationManager.setExpirationDate(to: Date())
```

Comment:
This test is valid in `master` and thus can be added separately

Reply:
I'm actually making some changes to these tests in `proxy-tokens-server`.

Reply:
As noted in the other related thread, I believe that even if small changes are needed, things that are valid outside of the PR should not be introduced by it. It just increases complexity. This MR can then focus on refining these by the amount that is required by other changes that are part of it